roles/tftp-server/files/late_command.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -euf
+
+# This script setups ansible and runs it
+# It should be ran at the end of the basic installation of a machine
+
+apt install -y ansible git eatmydata
+
+# We clone our ansible repository and copy the ansible config files
+
+git clone https://anonscm.debian.org/git/debconf-video/ansible.git /root/debconf-ansible
+git clone https://github.com/xfxf/lca2017-av.git /root/lca2017-av
```

Comment:
ah right.. 
I knew something wasn't quite right, but thought I was doing the equivalent thing just changing the host from github to local box.
but the whole late_command.sh name/url was a config option, and now that is gone.
Hmm...